app/forms/schools/add_participant_form.rb

```diff
@@ -275,5 +280,13 @@ def store_validation_result!(profile)
         },
       )
     end
+
+    def send_added_and_validated_email(profile)
+      ParticipantMailer.sit_has_added_and_validated_participant(participant_profile: profile, school_name: school_cohort.school.name)
+    end
+
+    def validation_record(profile)
+      ECFParticipantValidationData.find_by(participant_profile: profile)
+    end
```

Comment:
Is this worth pulling out do you think rather than just referencing directly?
ie. on line 268 `profile.ecf_participant_validation_data.present?`

Reply:
yeahhhh definitely, makes sense to pull it out